gotests/main.go

```diff
@@ -38,6 +40,7 @@ var (
 	exportedFuncs = flag.Bool("exported", false, `generate tests for exported functions and methods. Takes precedence over -only and -all`)
 	allFuncs      = flag.Bool("all", false, "generate tests for all functions and methods")
 	printInputs   = flag.Bool("i", false, "print test inputs in error messages")
+	subtests      = flag.Bool("subtests", false, "generate tests using the Go 1.7 subtests feature")
```

Comment:
I think we should enable this by default on Go 1.7. Subtests is one of the key features that makes table driven tests nice to use. As such, we should encourage its usage as much as possible, hence enabling it by default. 

We should call the flag `nosubtests` which defaults to false, but when provided disables subtests.